Cartfile.resolved

```diff
@@ -1 +1,2 @@
-github "krzyzanowskim/CryptoSwift" "0.2.2"
+github "krzyzanowskim/CryptoSwift" "0.2.3"
+github "jspahrsummers/xcconfigs" "0.9"
```

Comment:
I'd rather we kept it as it is. This adds additional complexity and multiple different files (25 of them) to track project settings. Would you be able to remove this from the pull request?